File: gin-vue-admin-main/server/router/wl_driver/wl_system_monior.go
package wl_driver

import (
	"github.com/flipped-aurora/gin-vue-admin/server/middleware"
	"github.com/gin-gonic/gin"
)

type WlSystemMoniorRouter struct{}

func (s *WlSystemMoniorRouter) InitWlSystemMoniorRouter(Router *gin.RouterGroup, PublicRouter *gin.RouterGroup) {
	wlSystemMoniorRouter := Router.Group("wlSystemMonior").Use(middleware.OperationRecord())
	wlSystemMoniorRouterWithoutRecord := Router.Group("wlSystemMonior")
	{
		wlSystemMoniorRouter.POST("createWlSystemMonior", wlSystemMoniorApi.CreateWlSystemMonior)
		wlSystemMoniorRouter.DELETE("deleteWlSystemMonior", wlSystemMoniorApi.DeleteWlSystemMonior)
		wlSystemMoniorRouter.DELETE("deleteWlSystemMoniorByIds", wlSystemMoniorApi.DeleteWlSystemMoniorByIds)
		wlSystemMoniorRouter.PUT("updateWlSystemMonior", wlSystemMoniorApi.UpdateWlSystemMonior)
	}
	{
		wlSystemMoniorRouterWithoutRecord.GET("findWlSystemMonior", wlSystemMoniorApi.FindWlSystemMonior)
		wlSystemMoniorRouterWithoutRecord.GET("getWlSystemMoniorList", wlSystemMoniorApi.GetWlSystemMoniorList)
	}
}

File: gin-vue-admin-main/web/src/api/wl_driver/wlSystemMonior.js
import service from '@/utils/request'

export const getWlSystemMoniorList = (params) => {
  return service({
    url: '/wlSystemMonior/getWlSystemMoniorList',
    method: 'get',
    params
  })
}

export const createWlSystemMonior = (data) => {
  return service({
    url: '/wlSystemMonior/createWlSystemMonior',
    method: 'post',
    data
  })
}

export const updateWlSystemMonior = (data) => {
  return service({
    url: '/wlSystemMonior/updateWlSystemMonior',
    method: 'put',
    data
  })
}

export const deleteWlSystemMonior = (params) => {
  return service({
    url: '/wlSystemMonior/deleteWlSystemMonior',
    method: 'delete',
    params
  })
}

export const deleteWlSystemMoniorByIds = (params) => {
  return service({
    url: '/wlSystemMonior/deleteWlSystemMoniorByIds',
    method: 'delete',
    params
  })
}

export const findWlSystemMonior = (params) => {
  return service({
    url: '/wlSystemMonior/findWlSystemMonior',
    method: 'get',
    params
  })
} 